test/parallel/test-single-executable-application.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const common = require('../common');
+
+// This tests the creation of a single executable application.
+
+const fixtures = require('../common/fixtures');
+const tmpdir = require('../common/tmpdir');
+const { copyFileSync, readFileSync, writeFileSync } = require('fs');
+const { execFileSync } = require('child_process');
+const { join } = require('path');
+const { strictEqual } = require('assert');
+
+if (!process.config.variables.single_executable_application)
+  common.skip('Single Executable Application support has been disabled.');
+
+if (!['darwin', 'win32', 'linux'].includes(process.platform))
```

Comment:
Sorry if I missed it, but does SEA work on all of the other platforms? I saw the note about testing under Linux support, but nothing specifying macOS, Windows, and Linux only.

Reply:
Yes, it works on all 3 - Windows, macOS and Linux

Reply:
I meant the other platforms like AIX, SmartOS, etc.

Reply:
Ah, those ones are not tested on CI because of problems during the injection phase from postject's side. If the resource is correctly injected into the binary, it might work.

Reply:
If the best we can claim is that "it might work" then I think we should document that only macOS, Windows, and specific AMD64 Ubuntu Linux are supported.